medicines/doc-index-updater/Cargo.toml

```diff
@@ -20,6 +20,7 @@ md5 = "0.7.0"
 redis = { version = "0.15.1", features = ["tokio-rt-core"] }
 regex = "1.3.1"
 reqwest = { version = "0.10.4", features = ["json"] }
+search_client =  { git = "https://github.com/MHRA/products", branch = "createable-deleteable-into-search-crate" }
```

Comment:
I think this is the problem that [Cargo workspaces][1] solve. It seems weird for a repo to reference itself like this.

[1]: https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html

Reply:
Ok cool, we'll try it and compare

Reply:
the issue with workspaces is that you then reference the search-client package by doing `search_client = { path = ../search-client }` and then `make docker-build` won't work as the search-client is below the root of where the Dockerfile is.   So we could change all that too and all the build/release process but potentially a lot of changes (maybe too much right now) to achieve this?

Reply:
changed to use tar to pipe both the `doc-index-updater` and the `search-client` into `docker build` (via `make docker-build`)